### ISSUE TITLE
Can't uninstall: ActiveRecord::IrreversibleMigration

### ISSUE THREAD

**Issue by ppoektos** [dhanasingh/redmine_wktime#128]
```
# rake redmine:plugins:migrate NAME=redmine_wktime VERSION=0 RAILS_ENV=production
Migrating redmine_wktime (ERPmine)...
== 11 CreateWkAssetManagement: reverting ======================================
rake aborted!
StandardError: An error has occurred, all later migrations canceled:

ActiveRecord::IrreversibleMigration
/usr/local/rvm/gems/ruby-1.9.3-p327/gems/activerecord-4.2.8/lib/active_record/migration/command_recorder.rb:65:in `inverse_of'
/usr/local/rvm/gems/ruby-1.9.3-p327/gems/activerecord-4.2.8/lib/active_record/migration/command_recorder.rb:50:in `record'
/usr/local/rvm/gems/ruby-1.9.3-p327/gems/activerecord-4.2.8/lib/active_record/migration/command_recorder.rb:83:in `execute'
/usr/local/rvm/gems/ruby-1.9.3-p327/gems/activerecord-4.2.8/lib/active_record/migration.rb:665:in `block in method_missing'
/usr/local/rvm/gems/ruby-1.9.3-p327/gems/activerecord-4.2.8/lib/active_record/migration.rb:634:in `block in say_with_time'
/usr/local/rvm/gems/ruby-1.9.3-p327/gems/activerecord-4.2.8/lib/active_record/migration.rb:634:in `say_with_time'
/usr/local/rvm/gems/ruby-1.9.3-p327/gems/activerecord-4.2.8/lib/active_record/migration.rb:654:in `method_missing'
/var/www/rm34/plugins/redmine_wktime/db/migrate/011_create_wk_asset_management.rb:51:in `change'
/usr/local/rvm/gems/ruby-1.9.3-p327/gems/activerecord-4.2.8/lib/active_record/migration.rb:606:in `block in exec_migration'
/usr/local/rvm/gems/ruby-1.9.3-p327/gems/activerecord-4.2.8/lib/active_record/migration.rb:495:in `block (2 levels) in revert'
/usr/local/rvm/gems/ruby-1.9.3-p327/gems/activerecord-4.2.8/lib/active_record/migration/command_recorder.rb:38:in `revert'
/usr/local/rvm/gems/ruby-1.9.3-p327/gems/activerecord-4.2.8/lib/active_record/migration.rb:495:in `block in revert'
/usr/local/rvm/gems/ruby-1.9.3-p327/gems/activerecord-4.2.8/lib/active_record/migration.rb:642:in `suppress_messages'
/usr/local/rvm/gems/ruby-1.9.3-p327/gems/activerecord-4.2.8/lib/active_record/migration.rb:494:in `revert'
/usr/local/rvm/gems/ruby-1.9.3-p327/gems/activerecord-4.2.8/lib/active_record/migration.rb:606:in `exec_migration'
/usr/local/rvm/gems/ruby-1.9.3-p327/gems/activerecord-4.2.8/lib/active_record/migration.rb:592:in `block (2 levels) in migrate'
/usr/local/rvm/gems/ruby-1.9.3-p327/gems/activerecord-4.2.8/lib/active_record/migration.rb:591:in `block in migrate'
/usr/local/rvm/gems/ruby-1.9.3-p327/gems/activerecord-4.2.8/lib/active_record/connection_adapters/abstract/connection_pool.rb:292:in `with_connection'
/usr/local/rvm/gems/ruby-1.9.3-p327/gems/activerecord-4.2.8/lib/active_record/migration.rb:590:in `migrate'
/usr/local/rvm/gems/ruby-1.9.3-p327/gems/activerecord-4.2.8/lib/active_record/migration.rb:768:in `migrate'
/usr/local/rvm/gems/ruby-1.9.3-p327/gems/activerecord-4.2.8/lib/active_record/migration.rb:998:in `block in execute_migration_in_transaction'
/usr/local/rvm/gems/ruby-1.9.3-p327/gems/activerecord-4.2.8/lib/active_record/migration.rb:1046:in `ddl_transaction'
/usr/local/rvm/gems/ruby-1.9.3-p327/gems/activerecord-4.2.8/lib/active_record/migration.rb:997:in `execute_migration_in_transaction'
/usr/local/rvm/gems/ruby-1.9.3-p327/gems/activerecord-4.2.8/lib/active_record/migration.rb:959:in `block in migrate'
/usr/local/rvm/gems/ruby-1.9.3-p327/gems/activerecord-4.2.8/lib/active_record/migration.rb:955:in `each'
/usr/local/rvm/gems/ruby-1.9.3-p327/gems/activerecord-4.2.8/lib/active_record/migration.rb:955:in `migrate'
/usr/local/rvm/gems/ruby-1.9.3-p327/gems/activerecord-4.2.8/lib/active_record/migration.rb:830:in `down'
/usr/local/rvm/gems/ruby-1.9.3-p327/gems/activerecord-4.2.8/lib/active_record/migration.rb:805:in `migrate'
/var/www/rm34/lib/redmine/plugin.rb:481:in `migrate_plugin'
/var/www/rm34/lib/redmine/plugin.rb:453:in `migrate'
/var/www/rm34/lib/redmine/plugin.rb:464:in `migrate'
/var/www/rm34/lib/tasks/redmine.rake:135:in `block (3 levels) in <top (required)>'
/usr/local/rvm/gems/ruby-1.9.3-p327/gems/rake-12.2.1/exe/rake:27:in `<top (required)>'
/usr/local/rvm/gems/ruby-1.9.3-p327/bin/ruby_executable_hooks:15:in `eval'
/usr/local/rvm/gems/ruby-1.9.3-p327/bin/ruby_executable_hooks:15:in `<main>'
Tasks: TOP => redmine:plugins:migrate
(See full trace by running task with --trace)
```

**Comment by dhanasingh** [comment]
We will take a look and get back to you.
In the mean time, you can manually remove all the tables starting with wk
and also remove all the versions ending with redmine_wktime from the schema_migrations table

**Comment by ppoektos** [comment]
Could you please provide an automate solution with smooth uninstallation?

**Comment by dhanasingh** [comment]
This is fixed in ERPmine 3.2, can you please check
